state_synchronizer/src/peer_manager.rs

```diff
@@ -42,7 +55,7 @@ impl PeerManager {
     pub fn set_peers(&mut self, peer_ids: Vec<PeerId>) {
         let new_peers = peer_ids
             .into_iter()
-            .map(|peer_id| (peer_id, PeerInfo::new(true, true)))
+            .map(|peer_id| (peer_id, PeerInfo::new(true, true, MAX_SCORE)))
```

Comment:
why do we set the "alive" field in PeerInfo to `true` here, while it's set to false in `::new()`?

Reply:
`set_peers` call is done mostly by Consensus client when it sets peers to list of validators that signed target QC. When consensus discovers new block, assumption is that signatures belong to alive nodes 
naming might be confusing. We should probably rename method